applications/crossbar/src/modules/provisioner_v5.erl

```diff
@@ -444,6 +436,22 @@ send_req('accounts_update', JObj, AuthToken, AccountId, _) ->
     Resp = ibrowse:send_req(UrlString, Headers, 'post', Data, HTTPOptions),
     handle_resp(Resp).
 
+payload('account', JObj, AccountId) ->
+    ResellerId = wh_services:find_reseller_id(AccountId),
+    Setters = [ fun (J) -> wh_json:set_value(<<"create_if_missing">>, true, J) end
+              , fun (J) -> wh_json:set_value(<<"reseller_id">>, ResellerId, J) end
+              , fun (J) -> wh_json:set_value(<<"data">>, JObj, J) end ],
```

Comment:
Please add the merge flag to the account update as well.

fun (J) -> wh_json:set_value(<<"merge">>, true, J) end